.github/workflows/_build_rosetta.yaml

```diff
@@ -1,13 +1,38 @@
 name: ~build Rosetta container
 
 on:
+  workflow_dispatch:
+    inputs:
+      BASE_IMAGE:
+        type: string
+        description: 'Base docker image that provides the foundation library:'
+        required: false
+        default: ghcr.io/nvidia/t5x:latest
+      BASE_LIBRARY:
```

Comment:
It seems that you are trying to accommodate the scenario where a user wants to build on top of an alternative image other than `ghcr.io/nvidia/${BASE_LIBRARY}:latest`. Meanwhile, we'd better perform some input validation so a user gets a meaningful error message when, say, trying to build a pax rosetta but forget to change the default image from t5x to pax.

How about we make `BASE_LIBRARY` as the first input and make `BASE_IMAGE` optional with an empty default? We can then add a job step to dynamically pick the default image, or check for the existence of t5x/pax in the user-specified container before starting the actual build.

Reply:
Good point. I can make that change